config/testdata/conf.good.yml

```diff
@@ -109,7 +109,8 @@ scrape_configs:
 - job_name: service-kubernetes
 
   kubernetes_sd_configs:
-  - api_servers:
+  - kind: service_endpoints
```

Comment:
Currently it's the most straightforward and easy-to-use one. Especially when wanting to have a straightforward job label.